### PR TITLE
800 802 gpg style tweaks

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
makes the animations nicer

### CODE REVIEW

File: shared/common-adapters/icon.desktop.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ export default class Icon extends Component {
       return <FontIcon
         title={this.props.hint}
         style={{...styles.icon, ...this.props.style}}
-        className={`fa ${iconType}`}
+        className={`fa ${iconType} ${this.props.className}`}
```
Comment (chrisnojima) [inline shared/common-adapters/icon.desktop.js]: we should talk about generally plumbing all these react base props through, maybe some utility

**Comment by chrisnojima** [comment]
zeplin removes whitespace stuff out of the exported assets so we're using a 1% white pixel to make our icons the same size so we don't need extra offsets in css. if you think you have to do that lets just fixup the assets

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**